openmeetings-web/src/main/webapp/css/raw-room.css

```
@@ -28,37 +16,44 @@
 	padding: 0;
 	margin: 0;
 }
-.room-block .sidebar {
+.room-block .sb-wb {
+	height: calc(100% - var(--room-menu-height));
+	width: calc(100% - var(--chat-width) - var(--buffer-size));
+	display: inline-block;
+	position: relative;
+}
+.room-block .sb-wb .sidebar {
 	min-width: 155px;
 	width: var(--room-sidebar-width);
 	position: absolute;
-	height: calc(100% - var(--room-menu-height));
+	height: 100%;
+	top: 0;
 	right: auto;
 	left: 0;
 }
-.room-block .sidebar #room-sidebar-tabs {
+.room-block .sb-wb .sidebar #room-sidebar-tabs {
```

This looks strange to me.
Could it not just be: #room-sidebar-tabs ? And ID should be always unique in the code.

```suggestion
#room-sidebar-tabs {
```
?

---

I would agree
BUT according to might experience: the more specific the rule - the less troubles :))

Can we do CSS optimizations and SASS after release? 
So we will have more time to tune and de-bug everything :))